auth-web/src/components/auth/account-settings/team-management/MemberDataTable.vue

```diff
@@ -141,8 +141,8 @@
       <!-- Leave Account -->
       <v-btn
         icon
-        :aria-label="canDissolve() ? 'Dissolve Account' : 'Leave Account'"
-        :title="canDissolve() ? 'Dissolve Account' : 'Leave Account'"
+        :aria-label="'Leave Account'"
```

Comment:
no need of :area-label instead can we write like this aria-label="Leave Account"

Reply:
Yup..good catch ..will change it